CONTRIBUTING.md

```diff
@@ -66,7 +66,7 @@ anybody starts working on it.
 We are always thrilled to receive pull requests. We do our best to process them
 quickly. If your pull request is not accepted on the first try,
 don't get discouraged! Our contributor's guide explains [the review process we
-use for simple changes](https://docs.docker.com/opensource/workflow/make-a-contribution/).
+use for simple changes](https://github.com/docker/docker/blob/master/project/REVIEWING.md).
```

Comment:
For other reviewers, I think this is a version of what this page was supposed to point at – https://web.archive.org/web/20160305022033/https://docs.docker.com/opensource/workflow/make-a-contribution/

Reply:
(I wonder if we still have something like these. These docs were good, and the replacements – although seemingly the closest we currently have – don't cover most of what they used to cover)

Reply:
I think most of what was left in the docs repo was moved back to https://github.com/moby/moby/tree/master/docs/contributing

Reply:
Yeah but did you see the link I posted? We don't really have a comparable doc right now – we have technical docs on how to debug daemon, setup environment, etc. but this had 
![Screenshot 2024-09-09 at 12 18 20](https://github.com/user-attachments/assets/e450a8d7-833f-4989-8819-f30ce92609be)